cook-book/getting-started/README.md

```diff
@@ -0,0 +1,287 @@
+
+Change [inventory](./hosts.yml):
+
+```diff
+@@ -23,6 +23,16 @@ all:
+         advertise_uri: '172.19.0.2:3302'
+         http_port: 8182
+
++    storage-1-replica-2:
++      config:
++        advertise_uri: '172.19.0.3:3304'
++        http_port: 8184
++
++    core-1:
++      config:
++        advertise_uri: '172.19.0.2:3301'
++        http_port: 8181
++
+   children:
+     # group instances by machines
+     machine_1:
+@@ -32,6 +42,7 @@ all:
+
+       hosts:
+         storage-1-replica:
++        core-1:
+
+     machine_2:
+       vars:
+@@ -40,12 +51,14 @@ all:
+
+       hosts:
+         storage-1:
++        storage-1-replica-2:
+
+     # group instances by replicasets
+     replicaset_storage_1:
+       hosts:
+         storage-1:
+         storage-1-replica:
++        storage-1-replica-2:
+       vars:
+         replicaset_alias: storage-1
+         failover_priority:
+@@ -53,3 +66,12 @@ all:
+           - storage-1-replica
+         roles:
+           - vshard-storage
++
++    replicaset_core_1:
++      hosts:
++        core-1:
++      vars:
++        replicaset_alias: core-1
++        roles:
++          - vshard-router
++          - failover-coordinator
+```
+
+### Configure instances
+
+To start and configure instance we can use
+[`configure_instances`](/doc/scenario.md#scenarios) pre-defined scenario.
+But it also contain steps we don't need (like `deliver_package` and `update_package`).
+`update_instance` step is used only with [multiversion approach](/doc/multiversion.md)
+, so it can be removed too.
+
+Use this `cartridge_scenario` in [playbook.configure-instances.yml](./playbook.configure-instances.yml):
+
+```yaml
+cartridge_scenario:
+  - configure_instance
+  - restart_instance
+  - wait_instance_started
+```
+
+Run new playbook:
+
+```bash
+ansible-playbook -i hosts.yml playbook.configure-instances.yml
+```
+
+Go to http://localhost:8182/admin/cluster/dashboard:
+
+![image](./images/instances-configured.png)
+
+Now we have two new unconfigured instances.
+
+### Accessing instances status and logs
+
+To check instance systemd service just connect to virtual machine and call
+`systemctl status`:
+
+```bash
+vagrant ssh vm1
+[vagrant@localhost ~]$ sudo systemctl status myapp@core-1
+● myapp@core-1.service - Tarantool Cartridge app myapp@core-1
+   Loaded: loaded (/etc/systemd/system/myapp@.service; enabled; vendor preset: disabled)
+   Active: active (running) since Fri 2021-04-02 17:33:14 UTC; 4min 33s ago
+  Process: 6413 ExecStartPre=/bin/sh -c mkdir -p /var/lib/tarantool/myapp.%i (code=exited, status=0/SUCCESS)
+ Main PID: 6414 (tarantool)
+   CGroup: /system.slice/system-myapp.slice/myapp@core-1.service
+           └─6414 tarantool init.lua: myapp@core-1
+```
+
+You can also use wildcard to get all `myapp` instance status:
+
+```bash
+sudo systemctl status myapp@*
+```
+
+Instances logs are available with `journalctl`.
+It can be helpful to pipe them to `less`:
+
+```bash
+sudo journalctl -u myapp@core-1 | less
```

Comment:
But `less` is used by default, isn't it?

Reply:
As I remember - not always.